fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/node/DoubleBindMarkerNode.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.node;
```

Comment:
Similar I'd say park this under a type  sub package

Reply:
#2321